common/system/tasks/cli.cpp

```diff
@@ -102,6 +104,7 @@ void cli_setDefaultConfig() {
   cliConfigs.groundElevationM = 0;
   cliConfigs.groundTemperatureC = 14.85;
   cliConfigs.radioChannel = 1;
+  snprintf(cliConfigs.callsign, sizeof(cliConfigs.callsign), "%s", defaultCall);
```

Comment:
I think based on https://cplusplus.com/reference/cstdio/snprintf/ this will null terminate the string unless you give it an 8 character long call sign, in which case it just discards stuff longer? For us that means that since the cli config struct is created and set to all zeros up there, you get. "KM6GNL\0" copied into {0,0,0,0,0,0,0,0}, and you don't necessarily deal with resetting the last byte. Should ground stations interpret the call sign as a c string, or as an array of characters?

(My idea would be to make the default call sign a char[] instead and just memset back to zeros/memcpy it in)

Reply:
[Established offline that 7-char callsign + null-terminator is fine]